src/workerd/api/worker-loader.c++

```diff
@@ -165,7 +165,7 @@ Worker::Script::Source WorkerLoader::extractSource(jsg::Lock& js, WorkerCode& co
 
         return {.name = entry.name, .content = [&]() -> Worker::Script::ModuleContent {
           KJ_IF_SOME(js, module.js) {
```

Comment:
It's probably worth adding a todo here to later apply the transpilation/stripping here as well. Certainly not needed for the initial pass but for dynamic worker loading it might be interesting to support later. /cc @kentonv